tests/kwik_e_mart/app_async.py

```diff
@@ -111,6 +111,13 @@
     responder.reply(['Bye', 'Goodbye', 'Have a nice day.'])
 
 
+@send_store_hours.handle(intent='find_nearest_store')
+async def transition_flows(request, responder):
+    del request
+    del responder
+    send_store_hours.reprocess()
+
+
```

Comment:
To test the case of badly specified reprocess, let's turn send_nearest_store into a dialogue_flow, with a handler there that will send a follow-on 'find_nearest_store' query into `reprocess('send_store_hours')`, and create a test that falls into that state.

Reply:
I decided against it since it seems really contrived.